virtcontainers/pkg/cgroups/manager.go

```diff
@@ -74,56 +73,25 @@ func UseSystemdCgroup() bool {
 
 // returns the list of devices that a hypervisor may need
 func hypervisorDevices() []specs.LinuxDeviceCgroup {
-	wildcard := int64(-1)
-	devicemapperMajor := int64(253)
-
 	devices := []specs.LinuxDeviceCgroup{}
 
-	devices = append(devices,
-		// hypervisor needs access to all devicemapper devices,
-		// since they can be hotplugged in the VM.
-		specs.LinuxDeviceCgroup{
```

Comment:
Another question I had was the reason for removing the devicemapper devices here?
What if the container was using devicemapper storage, will it be access the device node?

Reply:
> @devimc Another question I had was the reason for removing the devicemapper devices here?

device cgroup is updated before adding the device to the sandbox, so access to all the devicemapper devices is no longer needed

> What if the container was using devicemapper storage, will it be access the device node?

yes, when the devices node is added to the sandbox